packages/ui/src/components/PostScreenView.tsx

```diff
@@ -223,6 +223,8 @@ export function PostScreenView({
                   editPost={editPost}
                   channelType="chat"
                   getDraft={getDraft}
+                  showAttachmentButton={channel.type === 'chat'}
+                  showInlineAttachments={channel.type === 'chat'}
```

Comment:
this isn't worth fixing (might be doing a breaking change to this type soon), but leaving as a note in case someone wants it: this will not fix custom channels (I think you'd want to check `channel.contentConfiguration != null ? channel.contentConfiguration.defaultPostContentRenderer === 'tlon.r0.content.chat'` : channel.type === 'chat'`)